packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -117,17 +117,7 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
         }
 
         // Gets the datasets from the pattern or members of the dataset and displays any thrown errors
-        let responses: zowe.IZosFilesResponse[] = [];
-        responses = await Gui.withProgress(
-            {
-                location: vscode.ProgressLocation.Notification,
-                title: localize("getChildren.getDatasets.progress", "Get Dataset list command submitted."),
-            },
-            () => {
-                return this.getDatasets();
-            }
-        );
-
+        const responses = await this.getDatasets();
```

Comment:
Kind of nice to see how some of the logic get's simplified 🥳

Reply:
Going to look into the spinners because I would like to keep that UX consistent across tree views 😅

Reply:
@zFernand0 just an update: I noticed that the progress bar appears at the top of the tree view, and then if a few seconds go by without a response, then the spinner icon will appear. They seem to be implemented such that they will only show up if a pre-defined amount of time has passed. Furthermore, I was able to get the spinner to appear in the dataset view (at least for the session nodes), but not for the job nodes. I will look into this a bit more, but it seems like could be the way we are fetching jobs within that tree view.

![image](https://user-images.githubusercontent.com/86500639/219111913-31a1f3d1-ce4b-4b50-8c0d-d79ccdfdcf7f.png)

This is likely dependent on the amount of time taken to get a response from z/OSMF.